_includes/releases/v22.2/v22.2.1.md

```diff
@@ -0,0 +1,386 @@
+## v22.2.1
+
+Release Date: December 22, 2022
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v22-2-1-general-changes">General changes</h3>
+
+- CockroachDB Docker images are now multi-architecture manifests supporting the `x86_64` (amd64) and `arm64` architectures. [#90307][#90307]
+- Bulk operations now log the (sanitized) destinations they are connecting to. For example, `backup planning to connect to destination gs://test/backupadhoc?AUTH=specified&CREDENTIALS=redacted`. [#92208][#92208]
+
+<h3 id="v22-2-0-alpha-1-backward-incompatible-changes">Backward-incompatible changes</h3>
+
+- Changefeeds will now treat all errors, unless otherwise indicated, as [retryable errors](../v22.2/monitor-and-debug-changefeeds.html). [#92824][#92824]
```

Comment:
@amruss Here is the reference to the behavior change that links to the page where the reviewed PR will detail the change https://github.com/cockroachdb/docs/pull/15723

Reply:
Should this be under a "breaking changes" section? Or at least we should say changefeeds may not fail when they previously would, and make get stuck retrying

Reply:
OK, I've updated the description to include that. Throughout the docs "breaking change" == "backward-incompatible change", so I have kept the heading as-is for consistency.